RecoMuon/L2MuonSeedGenerator/src/L2MuonSeedGeneratorFromL1TkMu.h

```diff
@@ -0,0 +1,88 @@
+#ifndef RecoMuon_L2MuonSeedGenerator_L2MuonSeedGeneratorFromL1TkMu_H
+#define RecoMuon_L2MuonSeedGenerator_L2MuonSeedGeneratorFromL1TkMu_H
+
+/*  \class L2MuonSeedGeneratorFromL1TkMu
+ *
+ *   L2 muon seed generator:
+ *   Transform the L1TkMuon informations in seeds
+ *   for the L2 muon reconstruction
+ *   (mimicking L2MuonSeedGeneratorFromL1T)
+ *
+ *    Author: H. Kwon
+ *    Modified by M. Oh
+ */
+
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+// Data Formats
+#include "DataFormats/MuonSeed/interface/L2MuonTrajectorySeed.h"
+#include "DataFormats/MuonSeed/interface/L2MuonTrajectorySeedCollection.h"
+#include "DataFormats/TrajectoryState/interface/PTrajectoryStateOnDet.h"
+#include "DataFormats/MuonDetId/interface/DTChamberId.h"
+#include "DataFormats/MuonDetId/interface/CSCDetId.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/GeometrySurface/interface/BoundCylinder.h"
+#include "DataFormats/Math/interface/deltaR.h"
+#include "DataFormats/L1Trigger/interface/Muon.h"
+
+#include "DataFormats/L1TCorrelator/interface/TkMuon.h"
+#include "DataFormats/L1TCorrelator/interface/TkMuonFwd.h"
+
+#include "CLHEP/Vector/ThreeVector.h"
+
+#include "Geometry/CommonDetUnit/interface/GeomDetEnumerators.h"
+
+class MuonServiceProxy;
+class MeasurementEstimator;
+class TrajectorySeed;
+class TrajectoryStateOnSurface;
+
+namespace edm {
+  class ParameterSet;
+  class Event;
+  class EventSetup;
+}  // namespace edm
```

Comment:
rather than forward declaring these classes directly, it is better praxtice to include their forward declarations header:
```suggestion
#include "FWCore/Framework/interface/Frameworkfwd.h"
```